ldclient/requester.py

```diff
@@ -1,104 +1,44 @@
-                if inner.errno == errno.ECONNRESET and should_retry:
-                    log.warning(
-                        'ProtocolError exception caught while getting flag. Retrying.')
-                    return do_toggle(False)
-                else:
-                    log.exception(
-                        'Unhandled exception. Returning default value for flag.')
-                    return None
-            except Exception:
-                log.exception(
-                    'Unhandled exception. Returning default value for flag.')
-                return None
-
-        return callback(do_toggle(True))
+    def getAll(self):
+        hdrs = _headers(self._api_key)
+        uri = self._config.get_latest_features_uri
+        r = self._session.get(uri, headers=hdrs, timeout=(
+            self._config.connect, self._config.read_timeout))
+        r.raise_for_status()
+        features = r.json()
+        return features
 
-    def _toggle(self, key):
+    def get(self, key):
         hdrs = _headers(self._api_key)
-        uri = self._config.base_uri + '/api/eval/features/' + key
+        uri = self._config.get_latest_features_uri + '/' + key
         r = self._session.get(uri, headers=hdrs, timeout=(
-            self._config.connect, self._config.read))
+            self._config.connect, self._config.read_timeout))
         r.raise_for_status()
         feature = r.json()
         return feature
 
 
-class RequestsStreamProcessor(Thread, StreamProcessor):
-
-    def __init__(self, api_key, config, store):
-        Thread.__init__(self)
-        self.daemon = True
-        self._api_key = api_key
-        self._config = config
-        self._store = store
-        self._running = False
-
-    def run(self):
-        log.debug("Starting stream processor")
-        self._running = True
-        hdrs = _stream_headers(self._api_key)
-        uri = self._config.stream_uri + "/features"
-        messages = SSEClient(uri, verify=self._config.verify, headers=hdrs)
-        for msg in messages:
-            if not self._running:
-                break
-            self.process_message(self._store, msg)
-
-    def stop(self):
-        self._running = False
-
-    @staticmethod
-    def process_message(store, msg):
-        payload = json.loads(msg.data)
-        log.debug("Recieved stream event {}".format(msg.event))
-        if msg.event == 'put':
-            store.init(payload)
-        elif msg.event == 'patch':
-            key = payload['path'][1:]
-            feature = payload['data']
-            log.debug("Updating feature {}".format(key))
-            store.upsert(key, feature)
-        elif msg.event == 'delete':
-            key = payload['path'][1:]
-            # noinspection PyShadowingNames
-            version = payload['version']
-            store.delete(key, version)
-        else:
-            log.warning('Unhandled event in stream processor: ' + msg.event)
-
-
 class RequestsEventConsumer(Thread, EventConsumer):
```

Comment:
Can we move this class out to its own file? It's a separate concern from feature flag requests, and it's more in line with the other implementations to push this into a different file.